src/scheduler/agent/database.c

```diff
@@ -1098,11 +1098,17 @@ char* get_email_command(scheduler_t* scheduler, char* user_email)
         g_string_append_printf(client_cmd, " -S smtp-auth=plain");
       }
     }
+
     if(g_hash_table_contains(smtpvariables, "SMTPAuthUser"))
     {
-      g_string_append_printf(client_cmd, " -S smtp-auth-user=\"%s\" -S from=\"%s\"",
-          (char *)g_hash_table_lookup(smtpvariables, "SMTPAuthUser"),
-          (char *)g_hash_table_lookup(smtpvariables, "SMTPAuthUser"));
+     g_string_append_printf(client_cmd, " -S smtp-auth-user=\"%s\"" ,
+      (char *)g_hash_table_lookup(smtpvariables, "SMTPAuthUser"));
+    }
+
+    if(g_hash_table_contains(smtpvariables, "SMTPFrom"))
+    {
+     g_string_append_printf(client_cmd, " -S from=\"%s\"",
+     (char *)g_hash_table_lookup(smtpvariables, "SMTPFrom"));
```

Comment:
Please fix the indentation.

See [Coding Style](https://github.com/fossology/fossology/wiki/Coding-Style)